virttest/utils_test/libvirt.py

```diff
@@ -1592,6 +1592,22 @@ def alter_boot_order(vm_name, pci_id, boot_order=0):
     vmxml.sync()
 
 
+def wait_for_file_over(patt, xml_file, timeout=10):
```

Comment:
maybe put xml_file as first parameter,then patt as second

Reply:
I don't see any difference as both are needed without default value...

Reply:
functionally it is no difference , but from the  human natural  language understanding point,  first we have whole thing (xml_file),then we apply some rule (patt) on that.
But it just comments, doesn't block PR merge or not merge

Reply:
I think it should be personality issue. From my point of view, as "re" operation also set the pattern as first parameter so it is comfortable for me. Anyway, thanks for your comments